api/ReadableStream.json

```diff
@@ -379,7 +379,7 @@
               }
             ],
             "firefox_android": {
-              "version_added": false
+              "version_added": "mirror"
             },
```

Comment:
This should actually be the following:

```js
"firefox_android": "mirror"
```

(I can't write a suggestion for some odd reason...)

Reply:
THanks, fixed!